landscape.yml

```diff
@@ -486,38 +481,41 @@ landscape:
             logo: sumologic.svg
             crunchbase: https://www.crunchbase.com/organization/sumo-logic
   - category:
-    name: Change Management
+    name: Observability and Analysis
     subcategories:
       - subcategory:
-        name: Version Control
+        name: Measuring and Reporting
         items:
-          - item:
-            name: Git
-            homepage_url: https://git-scm.com/
-            repo_url: https://github.com/git/git
-            logo: git.svg
-            crunchbase: https://www.crunchbase.com/organization/git
-          - item:
-            name: Gitea
-            homepage_url: https://gitea.io/en-us
-            repo_url: https://github.com/go-gitea/gitea
-            logo: gitea.svg
-            twitter: https://twitter.com/giteaio
-            organization:
-              name: Gitea
-          - item:
-            name: GitHub
-            homepage_url: https://github.com/
-            logo: git-hub.svg
-            crunchbase: https://www.crunchbase.com/organization/github
+        - item:
+            name: Four Keys
+            homepage_url: https://cloud.google.com/blog/products/devops-sre/the-2019-accelerate-state-of-devops-elite-performance-productivity-and-scaling
+            repo_url: https://github.com/GoogleCloudPlatform/fourkeys
+            logo: key.svg
+            crunchbase: https://www.crunchbase.com/organization/google
       - subcategory:
-        name: Issue Tracking
+        name: Monitoring and Feedback Loop
         items:
-          - item:
-            name: Jira
-            homepage_url: https://www.atlassian.com/software/jira
-            logo: jira.svg
-            crunchbase: https://www.crunchbase.com/organization/atlassian
+        - item:
+            name: Dynatrace
+            homepage_url: https://www.dynatrace.com/
+            logo: dynatrace.svg
+            crunchbase: https://www.crunchbase.com/organization/dynatrace-software
+      - subcategory:
+        name: Tracing
+        items:
+        - item:
+            name: Honeycomb
+            homepage_url: https://www.honeycomb.io/
+            logo: honeycomb.svg
+            crunchbase: https://www.crunchbase.com/organization/honeycombio
+        - item:
+            name: OpenTelemetry
```

Comment:
@markkk Take out the 2 extra spaces at the start of the line and it should fix the build issue.